tesla_ble_mqtt/config.yaml

```diff
@@ -17,33 +17,25 @@ map:
   - share:rw
 startup: services
 options:
-  vin1: ""
-  vin2: ""
-  vin3: ""
-  presence_detection: false
-  ble_mac1: ""
-  ble_mac2: ""
-  ble_mac3: ""
+  vin_list: ""
+  ble_mac_list: ""
+  presence_detection_ttl: 180
   debug: false
-  mqtt_ip: ""
+  mqtt_server: ""
   mqtt_port: "1883"
-  mqtt_user: ""
-  mqtt_pwd: ""
+  mqtt_username: ""
+  mqtt_password: ""
   send_cmd_retry_delay: "5"
```

Comment:
this should be ble_cmd_retry_delay cuz send could be mosquitto or ?!

Reply:
Well it is ble_cmd now in the code.

Reply:
Don't put "5" - not needed. This will make the setting to show up only in the "additional section".